### PR TITLE
NR Dialog Rewrite/Edit/Touchup: 2-2 3, 4

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**
This PR is for the next three scenarios. They don't really have that much from the narrator, which is where I found most of the issues. Mostly it's just some general editing and adding a few exclamations when they're needed.

I had put descriptions of these changes in the comments of the changes, but then I lost them after I had to reconcile branches.

There also was a weird thing when VS Code decided to turn all the quotes into smart quotes and other formatting things like that. I think I got all of them fixed, but you'll have to see.

I think a lot of the edits are self-explanatory, if you want an explanation of why I did something, ask.

### CODE REVIEW

**Comment by TheShadowOfHassen** [comment]
Did it. Sorry. I didn't read the style guide.

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg [hunk 1]
```diff
@@ -446,13 +446,13 @@
...
                     speaker=Hamel
-                    message= _ "Bah! They look like fat slugs to me. Come on boys, let’s chop them to pieces before they can leave the canal."
+                    message= _ "Bah! They look like fat slugs to me. Come on, boys, let’s chop them to pieces before they can leave the canal!"
                 [/message]
             [/then]
             [else]
                 [message]
                     speaker=Hamel
-                    message= _ "Ghouls! They have an insatiable hunger for flesh and even one scratch from their claws will weaken you with sickness and disease. Nasty things; let’s chop them to pieces before they can leave the canal."
+                    message= _ "Ghouls! They have an insatiable hunger for flesh One scratch from their claws will weaken you with sickness and disease. Nasty things; let’s chop them to pieces before they can leave the canal."
```
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg:455]: Missing a full stop?

File: data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg [hunk 2]
```diff
@@ -139,12 +139,12 @@
...
         [message]
             speaker=Pruol
-            message= _ "Argh! They are stronger than we thought. Someone go tell the Master..."
+            message= _ "Argh! They are stronger than we thought. Someone, go tell the Master..."
         [/message]
 
         [message]
             role=Supporter
-            message= _ "What’s with this ‘Master’ business? It’s starting to make me nervous."
+            message= _ "What’s with this ’Master’ business? It’s starting to make me nervous."
```
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg:147]: I think the original left quote is correct, ditto for line 184.

File: data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg [hunk 3]
```diff
@@ -16,7 +16,7 @@
     [story]
         [part]
             music=traveling_minstrels.ogg
-            story= _ "Hoping to bypass the monsters which seemed to lurk at every corner in the caves, the humans and the dwarves returned to the surface hoping to speedily reach the mines. However, once they left the caves they found that they faced a new threat."
+            story= _ "Hoping to bypass the monsters which seemed to lurk at every corner in the caves, the humans and the dwarves returned to the surface in hope to speedily reach the mines. However, once they left the caves they found that they faced a new threat."
```
Comment (Wedge009) [inline data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg:19]: The original reads better to me - this change sounds awkward, not even sure I would consider it grammatically correct.

Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg]: Changed it back.

File: data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg (hunk 2 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg]: I think I fixed this.

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg (hunk 1 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg]: Yep I missed a period.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg [hunk 4]
```diff
@@ -15,7 +15,7 @@
 
     [story]
         [part]
-            story= _ "After reaching the dwarvish keep and tending to their wounded, the humans led by Tallin and the dwarves led by Lord Hamel hurried deeper into the caverns for a hasty council."
+            story= _ "After reaching the dwarvish keep and tending to their wounded the humans, led by Tallin, and the dwarves led by Lord Hamel hurried deeper into the caverns for a hasty council."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:18]: Removing that comma seems to be a grammatical error, the result doesn't scan at all to me.

The commas around "led by Tallin" don't add anything IMO, so in all respects I prefer the original string.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg [hunk 5]
```diff
@@ -130,22 +130,22 @@
...
+            message= _ "Oh, there are orcs enough down here, but ye’ll not likely see them; they hide from us like vermin. You see, lad, in these caves there is no more fearsome a foe than a dwarf. Not only do we move through the caves swiftly, but we know them like the backs of our hands. All those years ago, the orcs did indeed manage to swamp our defenses with their sheer numbers, but not before we surprised and slew their leader — that cursed Khazg Black-Tusk. Since then, although the orcs still foul these caves with their stench, their attacks have been uncoordinated and erratic, wi’ the orcs as likely to attack each other as they are to attack us."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:133]: I prefer the original for both changes.

The subject of the previous sentence is about dwarven movement points, so starting the new sentence with "The orcs" is a clear subject change. "The orcs did indeed" manages in 4 words to change subject and say that this will be an admission of something, so it's no surprise to have it qualified later in the sentence with "long years ago"; the new version is putting the timeframe before we know what the subject is.

It's possible "the cursed Black-Tusk" vs "that cursed Black-Tusk" is an en_GB vs en_US thing, but to me "that cursed Black-Tusk" would mean that the act of slaying their old leader put a curse on Black-Tusk.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg [hunk 6]
```diff
@@ -130,22 +130,22 @@
...
         [/message]
 
         [message]
             speaker=Hamel
-            message= _ "In more recent years, however, I ha’ noticed that there have been fewer orcs and more skeletons around. The orcs seem to fear them and shy away from them whenever possible, but their troll allies are another matter. Yon trolls actually seem to take great pleasure out of crushing the skeletons to the dust and sucking any marrow that remains in their bones..."
+            message= _ "In more recent years, however, I ha’ noticed that there have been fewer orcs and more skeletons around. The orcs seem to fear them and shy away from them whenever possible, but their troll allies are another matter. Yon trolls actually seem to take great pleasure out of crushing the skeletons to the dust and sucking what marrow remains in their bones."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:138]: I like the new one on this.

It's a small change in a long sentence, so having a translators' changelog would be helpful. An idea for how to do that was to have a line in the Git commit (as it's only useful to someone changing from old to new text) saying what changed.

Alternatively, po comments here in the WML file might be good for changes to strings that have been unchanged for ages (and this text seems to be last changed in 7d9ef83560f74c2258598337b6b842ddfa657c0f, 13 years ago).

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg [hunk 7]
```diff
@@ -170,7 +170,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "With all respect, my Lord Hamel, we the people from Dwarven Doors have faced certain death many times now. Rising up against the orcs was said to be certain death. Plunging headfirst into these bleak caves was said to be certain death. If our fate is death then so be it. Until that time comes, my people will fight and I will lead them to fight!"
+            message= _ "With all respect, my Lord Hamel, we the people from Dwarven Doors have faced certain death many times now. Rising up against the orcs was said to be certain death. Plunging headfirst into these bleak caves was said to be certain death. If our fate is death, then so be it. Until that time comes, my people will fight, and I will lead them."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:173]: The first added comma seems okay, but to me it doesn't need a comma there - it would have a slight pause there when spoken, yet it's a significant enough statement that the stress added by the reader is more important than the comma, and so it doesn't need the comma.

For the second sentence I prefer the original.

File: data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg [hunk 8]
```diff
@@ -139,7 +139,7 @@
 
         [message]
             speaker=Pruol
-            message= _ "Argh! They are stronger than we thought. Someone go tell the Master..."
+            message= _ "Argh! They are stronger than we thought. Someone, go tell the Master..."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg:142]: If it was a proper noun then I'd agree with the comma, but for "someone" I don't think it fits.

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg [hunk 9]
```diff
@@ -230,7 +230,7 @@
 
         [message]
             role=Supporter
-            message= _ "Grim gods of darkness! The whole place is swarming with undead! They have raised corpses to do their work."
+            message= _ "Grim gods of darkness! The whole place is swarming with undead! Look, they’re mining!"
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg:233]: The original is about horror and evil, setting up the enemy as bad, because necromancers have enslaved the dead. The new one misses that, and so I think the old one is better.

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg [hunk 10]
```diff
@@ -446,13 +446,13 @@
 
                 [message]
                     speaker=Hamel
-                    message= _ "Bah! They look like fat slugs to me. Come on boys, let’s chop them to pieces before they can leave the canal."
+                    message= _ "Bah! They look like fat slugs to me. Come on, boys, let’s chop them to pieces before they can leave the canal!"
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg:449]: The exclamation mark might be good, but the extra comma isn't.

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg [hunk 11]
```diff
@@ -446,13 +446,13 @@
...
                 [/message]
             [/then]
             [else]
                 [message]
                     speaker=Hamel
-                    message= _ "Ghouls! They have an insatiable hunger for flesh and even one scratch from their claws will weaken you with sickness and disease. Nasty things; let’s chop them to pieces before they can leave the canal."
+                    message= _ "Ghouls! They have an insatiable hunger for flesh. One scratch from their claws will weaken you with sickness and disease. Nasty things; let’s chop them to pieces before they can leave the canal."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg:455]: Both feel about the same to me, so I'd prefer "no change" over "change".

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg [hunk 12]
```diff
@@ -130,22 +130,22 @@
...
+            message= _ "What is the story with those skeletons anyway? They look like they were dwarves when they lived, but now they just seem to be mindless killers."
         [/message]
 
         [message]
             speaker=Hamel
-            message= _ "In truth, we ha’ no idea. They just started appearing one day some years after we were trapped here — by ones and twos in the beginning, and now by the hordes. We were fair frightened when we first encountered them, but we soon discovered they fell to our hammers just as readily as the orcs fell to our axes. So, it is not their existence I fear — for we have faced far worse trials — but their rising numbers gives cause for worry..."
+            message= _ "In truth, we ha’ no idea. They just started appearing one day. They came by ones and twos in the beginning, and now by the hordes. We were fair frightened when we first encountered them, but we soon discovered they fell to our hammers just as readily as the orcs fell to our axes. So, it is not their existence I fear — for we have faced far worse trials — but their rising numbers gives cause for worry..."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:148]: Old and new are about the same to me, so I'd prefer "no change".

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg [hunk 13]
```diff
@@ -130,22 +130,22 @@
...
+            message= _ "In more recent years, however, I ha’ noticed that there have been fewer orcs and more skeletons around. The orcs seem to fear them and shy away from them whenever possible, but their troll allies are another matter. Yon trolls actually seem to take great pleasure out of crushing the skeletons to the dust and sucking what marrow remains in their bones."
         [/message]
 
         [message]
             speaker=Tallin
-            message= _ "What is the story with those skeletons anyway? They look like they were dwarves when they lived, but now they just seem to be mindless killers, attacking anyone and anything."
+            message= _ "What is the story with those skeletons anyway? They look like they were dwarves when they lived, but now they just seem to be mindless killers."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:143]: I like this one.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg (hunk 5 above)
Comment (Wedge009) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:133]: I don't think it's a regional difference - at least not from my en_GB perspective. As a native English speaker I read the original as more suitable for a passive third-person perspective (ie external narrator), while the change seems more like a reference from a first-person speaker - which is the perspective given here. On the other hand, with the ambiguity of English, it could be read in the way that steve describes too. If that is a potential consequence of the change then perhaps it better to leave it as-is.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg (hunk 4 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg]: The first comma needs to come back, however the second makes a comma appositive and needs to be added about Lord Hamnel. I have no idea how I missed that.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg (hunk 7 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:173]: I honestly don't think the "to fight!" adds anything, nor does it sound natural. If someone else agrees with you though, I'll change it.

File: data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg (hunk 8 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg]: Right.

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg (hunk 10 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg]: OK.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg (hunk 12 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:148]: I thought the extra long sentence hard to follow, so I changed it.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg (hunk 13 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:143]: Thanks!

I find in my own writing and others that people write filler when they are thinking about what to say next. A good editing pass can prune those sentences.

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg (hunk 11 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg:455]: I thought the extra long sentence hard to follow, so I changed it. If someone else doesn't agree, I'll change it.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg (hunk 5 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg]: Unless Hamel is crazy and has a weird announcer voice, I'd say leave it. I don't see how you see how you can think it means they cursed Black-Tusk.

I added an explanation point at the end of the sentence. However, if it's still confusing we could change the cursed to another curse word: blasted, bloody?

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg (hunk 9 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg:233]: Who are they, though? If you can find a wording that has that adds the horror but makes it clear I'll add it.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg (hunk 6 above)
Comment (stevecotton) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:138]: The trouble is that different people have views of what's a "big" change. The main translation tool that works on Windows, Poedit, doesn't have diff highlighting (yet), and so translators end up comparing the text in the "Source text" pane with the "Previous source text" pane by hand. The proper fix is to get better translation tools, but at least for the moment I think this image explains why I'm being so hesitant about changes adding or removing commas.

image

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg (hunk 9 above)
Comment (stevecotton) [reply data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg:233]: I read it as an unspecified "they", meaning "whoever is running this place, even if we haven't seen them yet".

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg (hunk 6 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:138]: I've done it in all the other PRs, and they got merged. So... Maybe you just assume that the entire sentence need's translated?

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg (hunk 9 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg:233]: Couldn't they also assume that the undead have risen themselves?

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg [hunk 14]
```diff
@@ -446,13 +446,13 @@
 
                 [message]
                     speaker=Hamel
-                    message= _ "Bah! They look like fat slugs to me. Come on boys, let’s chop them to pieces before they can leave the canal."
+                    message= _ "Bah! They look like fat slugs to me. Come on, boys let’s chop them to pieces before they can leave the canal!"
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg:449]: The comma is actually wrong in your new one, please move it back to the original.

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg (hunk 11 above)
Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg:455]: I agree with the change in this case personally.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg (hunk 7 above)
[the remark this answers is not in the document]
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg]: That's much better than the old one.

File: data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg (hunk 9 above)
Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/04_Clearing_the_Mines.cfg]: FIxed it.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg [hunk 15]
```diff
@@ -170,7 +170,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "With all respect, my Lord Hamel, we the people from Dwarven Doors have faced certain death many times now. Rising up against the orcs was said to be certain death. Plunging headfirst into these bleak caves was said to be certain death. If our fate is death then so be it. Until that time comes, my people will fight and I will lead them to fight!"
+            message= _ "With all respect, my Lord Hamel, we the people from Dwarven Doors have faced certain death many times now. Rising up against the orcs was said to be certain death. Plunging headfirst into these bleak caves was said to be certain death. If that is our fate is death, so be it. Until that time comes, I will lead my people to fight for our freedom!"
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:173]: You forgot to remove the "is death" after fate.

Comment (TheShadowOfHassen) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg]: Fixed it.

**Review by nemaara — APPROVED** [review]
Okay from me, were there any other comments?

**Comment by TheShadowOfHassen** [comment]
From me, or other people?

I have the rest of the campaign edited. I can open the next PR if you want to leave this one for other people's thoughts.

**Comment by nemaara** [comment]
I mean I will leave it open for a few more days in case there were more comments that other people want to add on these revisions.

**Comment by TheShadowOfHassen** [comment]
Got it. Do you want me to start the next PR, so people can start commenting on it?

**Comment by nemaara** [comment]
Yes please, go ahead.

File: data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg [hunk 16]
```diff
@@ -15,7 +15,7 @@
 
     [story]
         [part]
-            story= _ "After reaching the dwarvish keep and tending to their wounded, the humans led by Tallin and the dwarves led by Lord Hamel hurried deeper into the caverns for a hasty council."
+            story= _ "After reaching the dwarvish keep and tending to their wounded, the humans led by Tallin, and the dwarves led by Lord Hamel, hurried deeper into the caverns for a hasty council."
```
Comment (stevecotton) [inline data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:18]: I still slightly prefer less commas here. The commas add pauses, but the action is that they're hurrying into a hasty meeting.

Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:18]: For me it's not about correctness or pauses but readability in this case. I feel like it's a bit hard to read with it entirely unbroken? Don't know if that helps you or not.

Comment (stevecotton) [reply data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg:18]: Okay, that sways me.

**Comment by TheShadowOfHassen** [comment]
@stevecotton Thanks!

Did that fix it?
